database/drivers/sqlite/sqlite.go

```diff
@@ -1,28 +1,32 @@
 package sqlite
 
 import (
+	"database/sql"
 	"path/filepath"
 
-	"github.com/jmoiron/sqlx"
 	// import sqlite3 driver
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/thrasher-corp/gocryptotrader/database"
 )
 
-// Connect creates a connection to the entered database
-// With SQLite the database is not created until first read/write
-
-func Connect() (*database.Database, error) {
-	if database.Conn.Config.Database == "" {
+// Connect opens a connection to sqlite database and returns a pointer to database.DB
+func Connect() (*database.Db, error) {
+	if database.DB.Config.Database == "" {
 		return nil, database.ErrNoDatabaseProvided
 	}
 
-	databaseFullLocation := filepath.Join(database.Conn.DataPath, database.Conn.Config.Database)
-	dbConn, err := sqlx.Open("sqlite3", databaseFullLocation)
+	x := database.DB.Config.Database[len(database.DB.Config.Database)-3 : len(database.DB.Config.Database)]
```

Comment:
I am considering dropping the forcing a .db extension completely after second thoughts i don't like the idea because technicality sqlite3 also supports db3, sqlite & sqlite3 extensions